src/TRoomDB.cpp

```diff
@@ -640,6 +641,7 @@ void TRoomDB::auditRooms(QHash<int, int>& roomRemapping, QHash<int, int>& areaRe
 
     // Check the set of area Ids against the ones we actually have:
     QSetIterator<int> itUsedArea(areaIdSet);
+    QList<int> missingAreasNeeded; // Later fix, forgot to handle wanted area ids that are +ve (and thus valid) but absent
```

Comment:
Actually, following on from the suggestion not to leave historic comments behind and to trust the git history, er, log, perhaps it isn't needed at all.